packages/app/src/app/pages/Patron/PricingModal/PricingChoice/index.js

```diff
@@ -21,63 +20,71 @@ import {
   StyledSignInButton,
 } from './elements';
 
-function PricingChoice({ store, signals, badge }) {
+const PricingChoice = ({ badge }) => {
+  const {
+    patron: {
+      priceChanged,
+      createSubscriptionClicked,
+      updateSubscriptionClicked,
+      cancelSubscriptionClicked,
+    },
+  } = useSignals();
+  const {
+    isLoggedIn,
+    isPatron,
+    user: { name, subscription },
```

Comment:
I prefer to do less destructuring here, currently it:

1. Removes the context of the property (`name` of what?)
2. Opens up the chance of bugs (what if `user` is undefined and only rendered under a condition?)

I think we should keep deep destructuring to a minimum, we can discuss it shortly later though. It's a quite small thing, so preferably we won't put much time into discussing it.